src/IO/Observer/WriteSimpleData.hpp

```diff
@@ -0,0 +1,69 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "IO/H5/AccessType.hpp"
+#include "IO/H5/File.hpp"
+#include "IO/Observer/ArrayComponentId.hpp"
+#include "IO/Observer/Tags.hpp"
+#include "Parallel/ConstGlobalCache.hpp"
+#include "Parallel/Info.hpp"
+#include "Parallel/Invoke.hpp"
+#include "Parallel/NodeLock.hpp"
+#include "Utilities/Requires.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace observers {
+namespace ThreadedActions {
+
+/*!
+ * \brief Append data to the volume h5 in a selected dataset.
```

Comment:
volume h5 file in a selected group?  (note don't use dataset as that has a specific meaning in H5)

Reply:
I thought I had intended the h5 sense of dataset -- I explicitly use `h5::Dat`, or are there other subtleties that I'm missing? I'd claim that the data written here is in a significantly different format from the volume data created by the other actions.

Reply:
How about `Append data to an h5::Dat subfile`? We would like to keep the `h5::*` code as an abstraction on top of the actual data layout so that users can refer to those classes if they need specific support or info on the data layout, and so we can more easily adjust the layout. We don't disagree it's in a different format, but we think the thing to highlight is using `h5::Dat` instead of `h5::VolumeData` rather than emphasizing how `h5::Dat` might be implemented

Reply:
Ah, gotcha; I'll change the comment to what you suggested.